minigalaxy/ui/gametile.py

```diff
@@ -381,19 +380,16 @@ def update_gtk_box_for_dlc(self, icon, title, installer):
             self.get_async_image_dlc_icon(icon, title)
         download_info = self.api.get_download_info(self.game, dlc_installers=installer)
         if self.game.is_installed(dlc_title=title):
-            icon_name = "emblem-default-symbolic.symbolic"
+            icon_name = "object-select"
             self.dlc_dict[title][0].set_sensitive(False)
         elif self.game.is_update_available(version_from_api=download_info["version"], dlc_title=title):
```

Comment:
I know this is not strictly icon thing, but can you switch places between this elif and its content and "if self.game.is_installed(dlc_title=title):"

Now if "if self.game.is_installed(dlc_title=title):" is true (game is installed), this elif is never checked, so update icon is never shown on DLC.

Reply:
This issue is now being addressed in #263